networks/kim_cnn.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from networks.base import BaseModel
+
+
+class KimCNN(BaseModel):
+    def __init__(self, config, embed_vecs):
+        super(KimCNN, self).__init__(config, embed_vecs)
+
+        self.filter_sizes = config.filter_sizes
+        emb_dim = embed_vecs.shape[1]
+        num_filter_maps = config.num_filter_maps
```

Comment:
"filter_maps" may not be a good name. I guess we can have
num_filters_per_size
and this should be put right after
filter_sizes
in yml file

Reply:
Solved.